components/list/list-item-drag-handle.js

```diff
@@ -221,15 +232,15 @@ class ListItemDragHandle extends LocalizeCoreElement(LitElement) {
 	}
 
 	_onInactiveKeyboard(e) {
-		if (e.type === 'click' || e.keyCode === keyCodes.ENTER || e.keyCode === keyCodes.SPACE || e.keyCode === keyCodes.LEFT) {
```

Comment:
This `LEFT` behaviour needed to be removed because `LEFT` and `RIGHT` are now being used as a way to move items into/out of nested lists.